prow/prow.yaml

```diff
@@ -989,4 +989,54 @@ spec:
       volumes:
       - name: prow-config
         configMap:
-          name: config                           
\ No newline at end of file
+          name: config
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: default
+  name: needs-rebase
+  labels:
+    app: needs-rebase
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: needs-rebase
+  template:
+    metadata:
+      labels:
+        app: needs-rebase
+    spec:
+      terminationGracePeriodSeconds: 180
+      containers:
+      - name: needs-rebase
+        image: gcr.io/k8s-prow/needs-rebase:v20190731-e3f7b9853
+        imagePullPolicy: Always
+        args:
+        - --dry-run=false
+        - --github-endpoint=http://ghproxy
```

Comment:
What is this endpoint?
I don't see any service called `ghproxy` running in the cluster?

Reply:
good point, I'll remove :sweat_smile:

Reply:
will you remove this?

Reply:
I will :joy_cat:

Reply:
done :heavy_check_mark:

Reply:
I still see it? 🙃

Reply:
huh for some reason… the `push`.. didn't happen :joy_cat:  or I pushed the wrong branch… or…

Reply:
thanks!